politech_backend/settings.py

```diff
@@ -45,6 +45,8 @@
     'rest_framework.authtoken',
     'main',
     'parliament',
+    'party',
```

Comment:
Nitpicking:
Could you sort the custom developed ones alphabetically?
Makes it easier to add apps in the future